po/es.po

```diff
@@ -0,0 +1,1212 @@
+"ahí."
+
+#: src\exploring.md:3
+msgid "What do you get?"
+msgstr "¿Y qué consigues?"
+
+#: src\exploring.md:21
+msgid "Is that docs?"
+msgstr "¿Eso es documentación?"
+
+#: src\exploring.md:35
+msgid "Also available online"
+msgstr "También disponible en línea"
+
+#: src\exploring.md:37
+msgid "See <https://public-docs.ferrocene.dev>"
+msgstr "Ver <https://public-docs.ferrocene.dev>"
+
+#: src\exploring.md:41
+msgid "![The docs page](./images/docs1.png)"
+msgstr "![La página de docs](./images/docs1.png)"
+
+#: src\exploring.md:45
+msgid "![The docs page](./images/docs2.png)"
+msgstr "![La página de docs](./images/docs1.png)"
+
+#: src\exploring.md:49
+msgid "![The docs page](./images/docs3.png)"
+msgstr "![La página de docs](./images/docs3.png)"
+
+#: src\writing.md:3
+msgid "The demo"
+msgstr "El demo"
+
+#: src\writing.md:5
+msgid "I have the `aarch64-unknown-none` target"
+msgstr "Tengo el target de `aarch64-unknown-none`"
+
+#: src\writing.md:6
+msgid "I want run to run a binary on an ARMv8-A machine in Aarch64 mode"
+msgstr "Quiero correr un binario en una máquina ARMv8-A en modo Aarch64"
+
+#: src\writing.md:7
+msgid "I'm using QEMU"
+msgstr "Estoy usando QEMU"
+
+#: src\writing.md:8
+msgid "I'll need a UART driver so you can see the output"
+msgstr "Necesitaré el driver de UART para que puedan ver el output"
+
+#: src\writing.md:9
+msgid "I'll need some assembly code to boot the chip"
+msgstr "Necesitaré algo de ensamblador para bootear el chip"
+
+#: src\writing.md:10
+msgid "All the demo source code is available"
+msgstr "Todo el código fuente del demo está disponible"
+
+#: src\writing.md:12
+msgid "Building the demo"
+msgstr "Construyendo el demo"
+
+#: src\writing.md:14
+msgid "You can build this with `cargo build`"
+msgstr "Lo puedes construir con `cargo build`"
+
+#: src\writing.md:15
+msgid "You can drive `rustc` manually with `$MY_FAVOURITE` build system"
+msgstr "Puedes ejecutar `rustc` manualmente con `$MI_SISTEMA_DE_BUILD_FAVORITO`"
+
+#: src\writing.md:16
+msgid "I show both"
+msgstr "Mostraré ambos"
+
+#: src\writing.md:18
+msgid "The boot.S file"
+msgstr "El archivo boot.S"
+
+#: src\writing.md:20
+msgid "Set the stack pointer, jump to a symbol called `kmain`."
+msgstr "Ponemos el pointer del stack, saltamos al símbolo llamado `kmain`."
+
+#: src\writing.md:36
+msgid "The skeleton"
+msgstr "El esqueleto"
+
+#: src\writing.md:38
+msgid ""
+"```rust [] ignore\n"
+"#![no_std]\n"
+"#![no_main]\n"
+"\n"
+"#[no_mangle]\n"
+"pub extern \"C\" fn kmain() {\n"
+"    todo!()\n"
+"}\n"
+"\n"
+"#[panic_handler]\n"
+"fn panic(_info: &core::panic::PanicInfo) -> ! { }\n"
+"```"
+msgstr ""
+
+#: src\writing.md:53
+msgid ""
+"We turned off name-mangling with `#[no_mangle]` so `boot.S` could jump to "
+"`kmain`. We also set the function to have `C` linkage (i.e. use a C "
+"compatible ABI), like assembly branch assumed."
+msgstr ""
+"Apagamos el name-mangling con `#[no_mangle]` para que `boot.S` pueda saltar a "
```

Comment:
I'd say "Desactivamos" instead of "Apagamos

Reply:
Cierto.